keyboards/keychron/k8/config.h

```diff
@@ -18,4 +16,8 @@
 
 #pragma once
 
-#include "config_common.h"
+// Enable RGB Matrix
+#define VIA_QMK_RGBLIGHT_ENABLE
+
+#define USB_POLLING_INTERVAL_MS 1
+#define QMK_KEYS_PER_SCAN 4
```

Comment:
should these be shipped by default?
should they go in a keymap/user directory?

Don't really know

Reply:
I do believe they where coming in from the xelus pr, and perhaps moved a bit around in my refactoring / rebase / merge efforts

Reply:
I removed the two..